### ISSUE TITLE
Tópicos Ago/2024

### ISSUE THREAD

**Issue by brenorb** [BitDevsBSB/BitDevsBSB#9]
1Bi canais Ln/ano
https://x.com/peterktodd/status/1813928457946153056

Provas ZK verificadas no Bitcoin pela primeira vez
https://bitcoinmagazine.com/technical/a-zero-knowledge-proof-is-verified-on-bitcoin-for-the-first-time-in-history

Dream Team de Op_codes para o Roasbeef
https://x.com/roasbeef/status/1813658193631023416

TCP over Nostr
https://primal.net/e/note1uk728fkpgt8qqe2a9gylc9af5e4kjkyr05f3c04zpv8jgzwu9s6qpks74v

Anúncio PR Review club

**Comment by luisschwab** [comment]
Taproot Assets on Lightning: Mainnet Release
https://lightning.engineering/posts/2024-07-23-taproot-assets-LN/

**Comment by luisschwab** [comment]
Nostr Web Services
https://github.com/asmogo/nws

**Comment by edilmedeiros** [comment]
Carteira de Bitcoin da Proton
https://proton.me/blog/proton-wallet-launch

**Comment by edilmedeiros** [comment]
A Bitaxe found a block
https://x.com/bitentrepreneur/status/1816173826754929125

**Comment by edilmedeiros** [comment]
BDK 1.0.0-beta.1
https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.1

**Comment by edilmedeiros** [comment]
Existe incentivo econômico em usar P2WSH ao invés de P2TR?
https://bitcoin.stackexchange.com/questions/123500/is-there-an-economic-incentive-to-use-p2wsh-over-p2tr

Bom tópico para iniciantes aprenderem que existem diferentes tipos de transações.

**Comment by edilmedeiros** [comment]
Por que os desevolvedores de protocolos trabalham para maximizar a receita dos mineradores?
https://bitcoin.stackexchange.com/questions/123679/why-do-protocol-developers-work-on-maximizing-miner-revenue

tl;dr não é para os mineradores, mas para os outros players da rede que não têm a mesma capacidade de prever os custos das transações.

**Comment by edilmedeiros** [comment]
Por que precisamos reestruturar a mempool para implementar o cluster mempool?
https://bitcoin.stackexchange.com/questions/123682/why-is-restructure-of-mempool-required-with-cluster-mempool

Mudanças de infraestrutura que parecem simples, mas são complexas.

**Comment by edilmedeiros** [comment]
Assinatura de transações com Yubikey
https://vxtwitter.com/Nneuman/status/1816862638900658562

**Comment by edilmedeiros** [comment]
BIP draft for the FROST threshold signing protocol
https://groups.google.com/g/bitcoindev/c/PeMp2HQl-H4

**Comment by brenorb** [comment]
yubikey signer 
https://twitter.com/Nneuman/status/1816862638900658562

**Comment by brenorb** [comment]
El-tor: High bandwidth Tor network, relays ganham sats
https://devpost.com/software/el-tor?ref_content=my-projects-tab&ref_feature=my_projects

**Comment by luisschwab** [comment]
Dark Skippy - a powerful new method for a malicious signing device to leak secret keys.
https://x.com/utxoclub/status/1820520960476561825

**Comment by luisschwab** [comment]
policy: enable full-rbf by default #30493 merged
https://github.com/bitcoin/bitcoin/pull/30493

**Comment by edilmedeiros** [comment]
Testnet 4 merged
https://github.com/bitcoin/bitcoin/pull/29775

**Comment by luisschwab** [comment]
NIST Releases First 3 Finalized Post-Quantum Encryption Standards
https://www.nist.gov/news-events/news/2024/08/nist-releases-first-3-finalized-post-quantum-encryption-standards

**Comment by brenorb** [comment]
Last mile solutions for LN
https://bitcoinmagazine.com/technical/assessing-the-lightning-networks-last-mile-solutions

**Comment by brenorb** [comment]
Engineering A backdoored hardware wallet 
https://t.co/uVUujoQzaz

**Comment by edilmedeiros** [comment]
Apple App Store censurando aplicativos de VPN (a mando do governo?)

https://x.com/protonvpn/status/1825181045035074016

**Comment by brenorb** [comment]
latest Ark release
https://x.com/ArkLabsHQ/status/1825542684553203722
